### ISSUE TITLE
 Add demo / screenshots

### ISSUE THREAD

**Issue by jtagcat** [gollum/gollum#1444]
When comparing [many wikis](https://github.com/awesome-selfhosted/awesome-selfhosted#wikis), one can't even start to compare, without first having to deploy the solution.

--
(EDIT by @dometto)
Screenshots at: https://github.com/gollum/gollum/wiki/Screenshots
We now have screenshots of:

- [x] Screenshots of how citations look
- [x] Emoji support
- [x] Screenshot of mathematics support through mathjax
- [x] Screenshot of CriticMarkup
- [x] Screenshot of general gollum features (editor, Overview, History, etc.)

**Comment by m4c0** [comment]
I would go further and ask to "eat your own food". It would be interesting if Gollum documentation is distributed via Gollum.

**Comment by dometto** [comment]
That's the plan for the upcoming 5.0 release (which is currently blocked on [this](https://github.com/libgit2/rugged/issues/818)), if we can find a free hosting service. Suggestions welcome!

**Comment by jtagcat** [comment]
I can host it, at [K-Space](https://k-space.ee). Though for that I would ask this to be packaged in a docker container. Set and forget + high availability.

**Comment by dometto** [comment]
@jtagcat that's a very friendly offer. Would it be an idea to create a sandbox that automatically resets the repository periodically, to avoid defacing?

@m4c0 on second thought, serving gollum's own documentation via gollum isn't as problem-free as I thought. On GitHub, users at least have to log in first before editing. A fully public facing wiki might result in the documentation becoming polluted, if people try out the wiki (this happens more than you think even on GitHub). Of course, we could set the documentation wiki to `--no-edit`, but then it wouldn't function as a full-fledged Demo.

All advise is welcome!

**Comment by dometto** [comment]
Also, if anyone who's been trying out the new 5.x branch would like to contribute some screenshots, we could start collecting them [here](https://github.com/gollum/gollum/wiki/Screenshots) for now. All help is much appreciated! We're very close to releasing 5.x now, and would like to get the documentation and readme in order for that.

(cc @jstolarek @TheCedarPrince)

**Comment by TheCedarPrince** [comment]
Hey @dometto - I would be happy to do some write-ups, but can you specifically tell me what you were thinking I could add? Here is what I am thinking I could add:

1. Screenshot of how citations look
2. Emoji support

I don't know what else you want me to add as I could also add snippets of .mkd or YAML depending on what you are thinking. Let me know.

**Comment by dometto** [comment]
@TheCedarPrince that's great! @bartkamphorst just made some suggestions for which features could use screenshots [here](https://github.com/gollum/gollum/pull/1486) Much appreciated!

**Comment by bartkamphorst** [comment]
> some suggestions for which features could use screenshots

To sum up here:

- [ ] Screenshots of how citations look
- [ ] Emoji support
- [ ] Screenshot of mathematics support through mathjax
- [ ] Screenshot of CriticMarkup

**Comment by jtagcat** [comment]
About automatic rollback/sandboxing:
I'd suggest having to have approval from contributors, like merge requests.

Though there probably is something to sandbox/rollback already (say containers, there's probably a specific thing built just to do this). I went through all wikis, checked them out, those without one docker command or demo, I made an issue and went on. Thus I currently have little interest in the project. I'm not willing to put in time to look in to the sandboxing.

**Comment by TheCedarPrince** [comment]
Hi @dometto and @bartkamphorst , here are two screenshots so far:

# Citations

This an example citation that uses the `ieee.csl` file -- it automatically generates all references according to the IEEE format (see bottom underneath **References**):

![image](https://user-images.githubusercontent.com/29561456/76886629-786bde00-6857-11ea-9334-0c4823ca4dd1.png)

Here is the associated markdown for that file:

```mkd
---
bibliography: zettel.bib
csl: ieee.csl
nocite: |
    
---

# WFDB Record

> **Date:** Tue 10 Mar 2020 06:15:25 AM UTC
> 
> **Summary:** Overview of the Waveform Database (WFDB) record format
>
> **Keywords:** ##summary #wfdb #dataformat #storage #healthcare #opensource #standard 

## **Note Linked To:**

### [The Technology Index](03162020223918-technology-index)
- **Summary:** This index is a compilation of utilities and resources when working with any sort of technology.

---

## **Overview:**

The Waveform Database (WFDB) Library is a record management system to handle already existing databases - it itself is not a database but a collection of tools to handle databases that follow the WFDB format. [@moodyWFDBProgrammerGuide2019] Furthermore, WFDB can be used for viewing, analyzing, and creating recordings of physiological signals [@garciamartinezLoadingPlottingFiltering2017].

## **How It Works:**

The way that the WFDB record format works is that it utilizes [@moodyWFDBProgrammerGuide2019]:

- **Header Files** to specify the characteristics of a signal (sampling frequency, etc.)
- **Annotation Files** to record annotations corresponding to a signal (generally kept in time with the rest of the signal)

## **Example Application - ECG Data:**

For example, when working with ECG data, the format utilizes the header and annotation files as follows [@moodyWFDBProgrammerGuide2019]:

- **Header Files** - could contain information about lead count, calibration format, etc.
- **Annotation Files** - could contain other information such as beat positions.

## **Miscellaneous Notes:**

- WFDB is considered a standard data format within the research world, and has found application in industry as well.
- The WFDB format is generally used for ECG records - however, it can extend to other data formats.
- A possible alternative to the WFDB record format is the European Data Format+ (EDF+) [@garciamartinezLoadingPlottingFiltering2017].

---
---

### **References:** 
```

# Emojis :smile: 

Here are an example of emojis in action:

![image](https://user-images.githubusercontent.com/29561456/76887098-1f507a00-6858-11ea-812f-b30ac7a3f5a0.png)

Here is the associated Markdown for this example:

```mkd
### **Overall rating:** 2️⃣

---
---

## **Goal Tracking:**

- Tea with God: ✅
- Read for 20 minutes: ✅
- Journal: ❌
- Work out: ⛔️

```

**Comment by TheCedarPrince** [comment]
Hope that helps! Thank you for the assistance on helping me run Gollum - it is a great project and I am glad I could contribute a little bit to it!

**Comment by dometto** [comment]
Many thanks @TheCedarPrince! In the first post of this issue there is now a checklist of screenshots.

**Comment by dometto** [comment]
![Untitled2](https://user-images.githubusercontent.com/147111/77331201-b1f18d00-6d20-11ea-91ba-733e130f79ae.gif)

**Comment by dometto** [comment]
@bartkamphorst could you add a screenshot of the CriticMarkup functionality? Then we're done for now I'd say

**Comment by bartkamphorst** [comment]
![Screen Shot 2020-03-23 at 17 53 01](https://user-images.githubusercontent.com/571173/77343164-8f1ba480-6d31-11ea-891f-b548345f387d.png)

**Comment by bartkamphorst** [comment]
@dometto I've added this screenshot to the wiki as well.

**Comment by dometto** [comment]
Great! Keeping this open for the live demo issue. More screenshots always welcome!

**Comment by dometto** [comment]
Closing in favor of #1599
